domain/notification.js

```diff
@@ -20,6 +22,29 @@ class Notification {
     return await this.notificationRepository.getNotification(this.notificationId);
   }
 
+  async sendNewNotificationEmail(userEmail) {
```

Comment:
You shouldn't copy/paste the sendEmail function. There should be only one function that sends an email that accepts an email and a message as a parameter. Inside these kind of functions that send emails, we should call that one function that will actually send the email